radis/test/misc/test_utils.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import print_function, absolute_import, division, unicode_literals
+import pytest
+from os.path import dirname, basename, join
+from os import listdir
+
+# Test to compare basenames of files whose names match the regular expression passed.
+def test_files_from_regex(*args, **kwargs):
+    from radis.misc.utils import get_files_from_regex
+    from radis.test.utils import getTestFile
+    from os.path import dirname, basename
+    import fnmatch
+
+    # path1 is for finding all the files with .par extension having regular expression *.par
+    path1 = getTestFile("*.par")
+
+    path_test = get_files_from_regex(path1)
+
+    # This list will have to be updated if more .par files are added/removed in/from the folder
+    path_actual = [
+        "hitran_co2_626_bandhead_4165_4200nm.par",
+        "hitran_2016_H2O_2iso_2000_2100cm.par",
+        "hitran_CO_fragment.par",
+        "hitran_co_3iso_2000_2300cm.par",
+        "hitran_CO2_fragment.par",
+    ]
+    path_actual.sort()
+    path_test.sort()
+
+    for i in range(
+        len(path_test)
+    ):  # finding basename because the function `get_file_from_regex` returns the absolute path
+        path_test[i] = basename(path_test[i])
+        assert path_test[i] == path_actual[i]
+
+    # path2 is for all the files with regular expression cdsd_*.par
```

Comment:
update the comment. Also, it's a nice information that explains what the function is doing: you should copy that in the function docstring